docs/all_development/roadmap/setting_up_dev_envt.rst

```diff
@@ -1,15 +0,0 @@
-=======================================
-Setting up your development environment
-=======================================
-
-You can locally develop Open edX code using two methods: `Devstack`_ and `Tutor`_. Devstack is used extensively by edX developers. They are both used widely by Open edX Community developers.
```

Comment:
**Reviewer Note:** These updated pointer to devstack is being reverted as part of this PR.